src/haddock/modules/analysis/caprieval/__init__.py

```diff
@@ -182,7 +183,24 @@ def _run(self) -> None:
         dump_weights(self.order)
 
         # Get reference file
-        references = self.get_reference(models)
+        try:
+            cgffversion = models[0].topology[0].rel_path.as_posix().split("_")[-1].split(".")[0]
+        except TypeError:
+            try:
+                cgffversion = models[0].topology.rel_path.as_posix().split("_")[-1].split(".")[0]
+            except AttributeError:
+                cgffversion = "aa"
+
+        if cgffversion == "martini2":
+            references = []
+            references = [
+                Path(martinize(ref_aa, self.path, False))
+                for ref_aa in self.get_reference(models)
+                ]
+            ff = cgffversion
+        else:
+            references = self.get_reference(models)
+            ff = "aa"
```

Comment:
Why using both `cgffversion` and `ff` if `ff = cgffversion` ?

```suggestion
        try:
            ff = models[0].topology[0].rel_path.as_posix().split("_")[-1].split(".")[0]
        except TypeError:
            try:
                ff = models[0].topology.rel_path.as_posix().split("_")[-1].split(".")[0]
            except AttributeError:
                ff = "aa"

        if ff == "martini2":
            references = [
                Path(martinize(ref_aa, self.path, False))
                for ref_aa in self.get_reference(models)
                ]
        else:
            references = self.get_reference(models)
```